client/v2/common/common.go

```diff
@@ -70,14 +70,13 @@ type InternalError error
 // extractError checks if the response signifies an error.
```

Comment:
This code is common to all endpoints. We now attempt to marshal the response body into the response body even if there was an error. Sometimes there is still useful information to return.